openapi-mock/package.json

```diff
@@ -16,22 +16,31 @@
     "express": "^4.16.4",
     "json-to-pretty-yaml": "^1.2.2",
     "morgan": "^1.9.1",
-    "path": "^0.12.7",
     "swagger-express-middleware": "^2.0.1",
-    "util": "^0.11.1",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "js-yaml": "^3.13.0"
   },
   "devDependencies": {
+    "@types/express": "^4.16.1",
+    "@types/js-yaml": "^3.12.0",
+    "@types/mocha": "^5.2.6",
+    "@types/morgan": "^1.7.35",
+    "@types/supertest": "^2.0.7",
```

Comment:
Why you have supertest as a dependency with and without types and in different versions?
Why there is a dependency to the swagger-express-middleware? In an old version

Reply:
`@types` are declaration libraries used by typescript's Intellisense. A package can choose to publish it's own declarations with the package itself (like us) or they can publish their declarations in a separate package to `@types` scope. Those packages only include declarations and no code. It is not mandatory to publish them every time with the real package so they might have a different version.

Reply:
Thanks for the explanation, well done :)